stepfunctions/stepfunctions/functions/delete_alias/index.py

```diff
@@ -0,0 +1,26 @@
+import json
+import boto3
+import uuid
+import os
+
+def handler(event, context):
+    # TODO implement
+
+    #pass in from step function but for now
+    
+    agent_id = event["agent_id"]
+    agent_alias_id = event["agent_alias_id"]
+    
+    bedrock_agent = boto3.client('bedrock-agent')
+    
+    response = bedrock_agent.delete_agent_alias(
```

Comment:
check the status of `response` and log error if needed before sending back `200`.